backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -218,9 +218,8 @@ export class ExperimentAssignmentService {
       where: {
         site: site,
         target: target,
-        user: userId,
+        user: userDoc,
```

Comment:
It seems like userId should be faster if TypeORM just needs the Primary Key, right? It may not really matter one way or another, just curious.